adoc/chapters/programming_interface.adoc

```diff
@@ -12808,6 +12809,41 @@ operation, as described in <<table.operators.reducer>>.
 include::{header_dir}/reducer.h[lines=4..-1]
 ----
 
+[[table.types.reducer]]
+.Member types and constants of the [code]#reducer# class
+[width="100%",options="header",separator="@",cols="50%,50%"]
+|====
+@ Member @ Description
+a@
+[source]
+----
+value_type
+----
+   a@ The data type of the reduction variable.  If this reducer object was
+      created from a buffer type [code]#BufferT#, this type is
+      [code]#BufferT::value_type#.  If this reducer object was created from a
+      USM pointer [code]#T*# or a span [code]#span<T, Extent>#, this type is
+      [code]#T#.
+
+a@
+[source]
+----
+binary_operation
+----
+   a@ The type of the combiner operator [code]#BinaryOperation# that was
+      passed to the reduction function that created this reducer object.
+
+a@
+[source]
+----
+static constexpr int dimensions
+----
+   a@ The number of dimensions of the reduction variable.  If this reducer
```

Comment:
Sorry, Side question. If the reduction variable was created using a `nd_item` it will take the dimension of the `nd_item`?

Reply:
No, I don't think so.  I believe the number of dimensions of the reduction variable is unrelated to the number of dimensions of the parallel_for.